### DISCUSSION TITLE
Car/bike icon

### DISCUSSION THREAD

**Discussion by danilopolani** [tailwindlabs/heroicons#400] [Ideas]
I found a `truck` icon but would nice also car and bike, thanks!

**Comment by darkmastermindz** [comment]
Duplicate of #443

**Reply by grantholle** [reply]
How is this (#400) a duplicate of #443, when #443 was created _after_ this one?

**Comment by livelo-app** [comment]
It would be great to have this feature! 😇

**Comment by komposer-aml** [comment]
It's 2023 - We still don't have a car?

**Comment by nonameolsson** [comment]
We are probably switching away from Heroicons due to the lack of a car icon 😔

**Comment by WebKenth** [comment]
What about a Road Icon? Stop Signs and the like? easy addition

**Comment by larshei** [comment]
A Car icon would be great.

In general, transport icons would be nice!
I'd be very happy about a car icon and any public transport icon (like a bus stop, bus, or train)

**Comment by tanguylegazon** [comment]
Please. This is the umpteenth time I've needed it. 😢

EDIT: Judging by the Discussions, this repo is pretty much dead. I think I can look elsewhere for transport icons... It's a shame, I don't like any of the alternatives as much.

**Reply by Abdulrahman-Al-Hasbani** [reply]
Hey, did you by any chance find a good alternative that you could recommend?

**Comment by Kouler** [comment]
heroicons doesn't give a f**k about you, guys - "We added a truck icon, so, that's enough"
